network/p2p/unicast/cache/unicastConfigCache.go

```diff
@@ -122,10 +123,26 @@ func (d *UnicastConfigCache) GetWithInit(peerID peer.ID) (*unicast.Config, error
 			EntityId: entityId,
 		}
 	}
-	entity, ok := d.peerCache.GetWithInit(entityId, initFunc)
-	if !ok {
-		return nil, fmt.Errorf("get or init for unicast config for peer %s failed", peerID)
+
+	// TODO: UnicastConfigCache implementation will be updated according to new usages in issue https://github.com/onflow/flow-go/issues/7076,
```

Comment:
```suggestion
	// TODO(malleability, #7076): UnicastConfigCache implementation will be updated according to new usages.
```

This format of TODO, allows quick filtering by topic and has the issue attached.

Reply:
[88006a5](https://github.com/onflow/flow-go/pull/7098/commits/88006a547d8578cd235697786004d3b10529a0d8)